sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -686,6 +687,17 @@ object SQLConf {
     .intConf
     .createWithDefault(100)
 
+  val CODEGEN_OBJECT_FALLBACK = buildConf("spark.sql.test.codegenObject.fallback")
```

Comment:
`CODEGEN_FACTORY_MODE` and `spark.sql.codegen.factoryMode`?

Reply:
hmm, shall we use `CODEGEN_FACTORY_MODE` or `CODEGEN_FALLBACK_MODE`?

Reply:
`CODEGEN_FACTORY_MODE` seems more accurate to me. This doesn't control all codegen fallback.